### ISSUE TITLE
Recommendation to adjust language in introduction of WCAG 3.0 to go beyond web technology

### ISSUE THREAD

**Issue by backwardok** [w3c/silver#563]
In the introduction of [WCAG 3.0](https://www.w3.org/TR/wcag-3.0/), there's the following content:

> These guidelines address accessibility of web content on desktops, laptops, tablets, mobile devices, wearable devices, and other web of things devices. They address various types of web content including static content, interactive content, visual and auditory media, and virtual and augmented reality. The guidelines also address related web tools such as user agents (browsers and assistive technologies), content management systems, authoring tools, and testing tools.

This language focuses specifically on web content, which makes it seem like it doesn't apply to something like a native mobile app. Even though WCAG 3.0 (and previous versions of WCAG) include guidelines and success criteria that apply to native apps as well, it's unclear from the language that this content is relevant to native applications. 

There was a conversation in a recent accessibility meetup around what qualifies as "web content", where there wasn't consensus on whether or not "web content" includes a native app or not. In my opinion, web content relates to something that uses web technology to display content available on the internet in some way. Given that some native apps do not require access to the internet, they don't seem like they fall strictly under "web content".

Could the language be updated to better indicate that it can apply outside of web content as well, if that's the intention?

For example:
> These guidelines address accessibility of digital content available on desktops, laptops, tablets, mobile devices, wearable devices, and other similar devices. They address various types of digital content, including static content, interactive content, visual and auditory media, and virtual and augmented reality. The guidelines also address related digital tools such as user agents (browsers and assistive technologies), content management systems, authoring tools, and testing tools.

**Comment by GreggVan** [comment]
Yes

I found in both reading and writing that it was confusing when it said  ‘guidelines’.  Did guidelines mean the WCAG 3 Guidelines  or the guidelines within the  WCAG3.

For example 
The guidelines will not contain normative material.
The guidelines will contain normative provisions.

Are both true.   

The first is the guidelines inside WCAG3
The second IS WCAG3. 

So the suggestion was that when referring to WCAG3 we do not refer to them/it  as  “the guidelines”  but rather as WCAG3
- and use the term “there guidelines’ to refer to the guidelines in WCAG3.
(It would also differentiate it from WCAG2x and other guidelines when quoted out of context. )

Best.

Gregg

**Comment by backwardok** [comment]
Disambiguating the term "guidelines" is also probably worthwhile, but I do want to mention that the intent behind this issue as filed is around the usage of the term "web" and suggesting something like the term "digital" to be more inclusive of mobile apps (if WCAG 3 is meant to cover mobile apps more intentionally)

**Comment by GreggVan** [comment]
Good point 
Have we settled on WCAG  or  are we still looking for a final name  -  including WAG  or something  - that looks like it might be different than just web content?

DAG  (digital accessibility guidelines?) 

PS if we wander outside of web — we should be aware that we open the scope of what we must address quite dramatically.  Web apps are different from mobile apps.  Our web guidelines cover a lot of ground in mobile apps — but there is a good bit more ground we have to cover if we wander outside of a browser  (i.e. if we wander outside of a mobile view of web content)…

**Comment by bruce-usab** [comment]
> Have we settled on WCAG

Yes, and that was a huge fight.  W3C Accessibility Guidelines, so "web content" dropped.

With teaching 2.x, I found that some people tripped over "guidelines" being the middle level of the hierarchy between Principal and Success Criteria.  So I agree it something worth being mindful of, but not a big deal.

**Comment by alastc** [comment]
I think the next step is a short update to the introduction, taking account of Gregg's comment above.

**Comment by alastc** [comment]
Ah, noticing that the introduction in discussion is for WCAG 3 not the requirements, so tagging appropriately and removing from the project.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
